### PR TITLE
Feature/update pom

### PR DESCRIPTION
**Pull request by jweiser**
Reorganize and streamline the POM file. All details are in the commit messages.

### CODE REVIEW

File: src/test/java/org/reactome/release/BioModelsUtilitiesTester.java [hunk 1]
```diff
@@ -6,21 +6,16 @@
...
 
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.*;
 
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({BioModelsUtilities.class})
-@PowerMockIgnore({"org.apache.logging.log4j.*", "javax.management.*", "javax.script.*",
-        "javax.xml.*", "com.sun.org.apache.xerces.*", "org.xml.sax.*", "com.sun.xml.*", "org.w3c.dom.*"})
+@RunWith(MockitoJUnitRunner.class)
```
Comment (SolomonShorser-OICR) [inline src/test/java/org/reactome/release/BioModelsUtilitiesTester.java:18]: These changes have all been tested, right?

Comment (jweiser) [reply src/test/java/org/reactome/release/BioModelsUtilitiesTester.java:18]: Yup, I ran the tests locally and they all still passed with the changes.

File: src/test/java/org/reactome/release/ModelsTSVParserTester.java [hunk 2]
```diff
@@ -81,7 +81,7 @@ public void testFileContainsCorrectMapping() {
         Map<String, Set<String>> pathwayToBioModelsIds = ModelsTSVParser.parse(filepath);
 
         // Tests pathway contains correct BioModels id
-        assertThat(pathwayToBioModelsIds.get("R-HSA-199418"), contains("BIOMD0000000175"));
+        assertThat(pathwayToBioModelsIds.get("R-HSA-199418"), hasItem("BIOMD0000000175"));
```
Comment (SolomonShorser-OICR) [inline src/test/java/org/reactome/release/ModelsTSVParserTester.java:84]: I'm curious why this was done. What's wrong with `contains`?

Comment (jweiser) [reply src/test/java/org/reactome/release/ModelsTSVParserTester.java:84]: `contains` requires all elements in the collection be specified in order to match where `hasItem` matches for any of the items.

R-HSA_199418 has more than one BioModels identifier mapped to it, so `hasItem` is needed to match a single one.  Alternatively, we could provide all identifiers and keep `contains`.  That might be a good way to go, too.

**Review by SolomonShorser-OICR — APPROVED** [review]

File: pom.xml [hunk 3]
```diff
@@ -1,30 +1,99 @@
...
     <artifactId>biomodels</artifactId>
-    <packaging>jar</packaging>
     <version>1.0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
     <name>biomodels</name>
-    <url>http://maven.apache.org</url>
+    <description>
+        Insertion of BioModels identifiers cross-references
+        (data generated by https://github.com/reactome/biomodels-mapper) for Pathway instances
```
Comment (cookersjs) [inline pom.xml:13]: Is there another tag that denotes dependencies/repositories used, rather then needing to cite in the description in parantheses?

File: pom.xml [hunk 4]
```diff
@@ -36,43 +105,39 @@
                     </descriptorRefs>
                 </configuration>
             </plugin>
+
+            <!-- allows 'mvn release' command to create a new release of the project -->
```
Comment (cookersjs) [inline pom.xml]: Do we have documentation somewhere that describes how to do releases this way?

Comment (SolomonShorser-OICR) [reply pom.xml]: Standard documentation for the maven release plugin is here: https://maven.apache.org/maven-release/maven-release-plugin/

File: pom.xml (hunk 3 above)
Comment (jweiser) [reply pom.xml:13]: I'm not sure I understand what you mean by another tag?

File: pom.xml (hunk 4 above)
Comment (jweiser) [reply pom.xml]: I can embed the URL in the comment just so it's easy to lookup how to use the command and for anyone new to the project and maven.

Comment (jweiser) [reply pom.xml]: Done: https://github.com/reactome/release-biomodels/pull/1/commits/3410122180d2c40637f273a34ae9078efec1387f

**Review by cookersjs — APPROVED** [review]

File: pom.xml (hunk 3 above)
Comment (jweiser) [reply pom.xml:13]: It looks like the `parent` and `dependency` tags are used for relationships for other repos, but I don't see any tags that discuss related or dependent repositories not directly used by the source of the current project.

**Pull request merged**